config/routes.rb

```diff
@@ -394,22 +394,26 @@
       get '/in_person/:step' => 'in_person#show', as: :in_person_step
       put '/in_person/:step' => 'in_person#update'
 
-      # deprecated routes
-      get '/confirmations' => 'personal_key#show'
-      post '/confirmations' => 'personal_key#update'
-    end
+      get '/by_mail' => 'gpo_verify#index', as: :gpo_verify
+      post '/by_mail' => 'gpo_verify#create'
```

Comment:
We don't need an alias for this route, right?

Reply:
`idv_gpo_verify_(path|url)` is used in a bunch of places. I don't think we need to do a separate alias to the POST version

Reply:
yup, as long as the URL is the same for both verbs, we only need the one alias